templates/newsletter.html

```diff
@@ -1,11 +1,16 @@
 <section data-component="n-newsletter-signup" data-newsletter-id="{{id}}" data-newsletter-name="{{name}}" class="n-newsletter-signup" {{#if isPremium}}data-newsletter-is-premium{{/if}}>
 	<div class="n-newsletter-signup__header">
-		<span class="n-newsletter-signup__heading">{{name}}<span class="n-util-visually-hidden">{{#if isPremium}}, premium{{/if}}{{#if frequency}}, {{frequency}}{{/if}}</span></span>
-		{{#if isPremium}}<span aria-hidden="true" class="n-newsletter-signup__top-meta o-labels o-labels--premium">Premium</span>{{/if}}
+		<div class="n-newsletter-signup__heading">
+			<div class="n-newsletter-signup__heading__mail-icon__wrapper">
+				<span class="n-newsletter-signup__heading__mail-icon"/>
+			</div>
+			<span class="n-newsletter-signup__heading__title">{{name}} Email<span class="n-util-visually-hidden">{{#if isPremium}}, premium{{/if}}{{#if frequency}}, {{frequency}}{{/if}}</span></span>
```

Comment:
@dapenduin Thank you for your review. I have changed the tag from `span` to `h2` and I am going to write a caution note on README for users about this template.